tests/unit/test_parser.py

```diff
@@ -142,3 +172,55 @@ def test_parse_state(self) -> None:
                 }
             }
         }
+
+    def test_parse_config_with_extra(self) -> None:
+        parser = RootTestParserWithExtra(
+            dm, test_data, config=True, state=False, extra={"os_version": "test"}
+        )
+        parsed_obj = parser.process()
+        assert parsed_obj.raw_value() == {
```

Comment:
It looks like this payload (lines 181-200) and the payload used in `.test_parse_state_with_extra()` (lines 207-226) are identical except for the `config` and `state` words. Would a variable fixture be better here to reduce duplication and improve readability?

Reply:
Not a fixture but a global variable like `test_data` might help. Will do.

Reply:
fixed